content/applications/inventory_and_mrp/manufacturing/workflows/oee.rst

```diff
@@ -0,0 +1,168 @@
+===============================
+Overall equipment effectiveness
+===============================
+
+.. |MO| replace:: :abbr:`MO (manufacturing order)`
+.. |OEE| replace:: :abbr:`OEE (overall equipment effectiveness)`
+.. |BoM| replace:: :abbr:`BoM (Bill of Materials)`
+
+In Odoo's *Manufacturing* app, overall equipment effectiveness (OEE) represents the amount of time a
+work center is *fully productive*. |OEE| is displayed as a percentage of the total time a work
+center is active.
+
+Fully productive time is considered to be time when the work center is operational **and**
+processing work orders that have not exceeded their *expected duration*.
+
+OEE helps manufacturing teams understand the efficiency of work centers, and the causes of
+manufacturing downtime.
+
+.. important::
+   Since |OEE| tracks work center productivity, using it requires enabling work centers in the
+   settings of the manufacturing app.
+
+   To do so, navigate to :menuselection:`Manufacturing app --> Configuration --> Settings`, and tick
+   the checkbox next to :guilabel:`Work Orders`, under the :guilabel:`Operations` heading.
+
+Efficiency standards
+====================
+
+For |OEE| to accurately reflect the percentage of fully productive time for a work center, the work
+center must be properly configured with the correct productivity metrics. These include the work
+center's *time efficiency*, *capacity*, and *|OEE| target*.
+
+Time efficiency
+---------------
+
+Time efficiency represents the efficiency of a work center when processing work orders, and is
+represented as a percentage. A time efficiency value of 100% signifies that the work center
+processes work orders at the speed of the expected duration listed on a product's |BoM|. A value
+less than or greater than 100% signifies that the work center processes work orders slower or faster
```

Comment:
```suggestion
represented as a percentage. A time efficiency value of *100%* signifies that the work center
processes work orders at the speed of the *expected duration* listed on a product's |BoM|. A value
*less than or greater than 100%* signifies that the work center processes work orders *slower or faster*
```
Optional formatting for emphasizing the connection between time efficiency value and work order speed.